.github/workflows/contrib.yml

```diff
@@ -25,14 +25,12 @@ jobs:
       run: |
         cd src/doc/contrib
         mdbook build
-        git worktree add gh-pages gh-pages
+        # Override previous ref to avoid keeping history.
+        git worktree add --orphan -B gh-pages gh-pages
```

Comment:
`--orphan` in `git worktree add` is a relatively new feature added in Git 2.42. I just confirmed that [GitHub hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#preinstalled-software) already pre-installed that version of Git. 👍🏾